### ISSUE TITLE
32953541: WKWebview accessibility navigation doesn't work when nested in UICollectionView or UIScrollView

### ISSUE THREAD

**Issue by openradar-mirror** [lionheart/openradar-mirror#17854]
#### Description

Summary:
VoiceOver navigation not working in nested WKWebView:
Swipe navigation of WKWebView inside of UICollectionViewCell doesn't escape to remaining accessibility elements (next/previous cell)
When nested in UIScrollView, swipe navigation works as expected; direct touch fails to grant focus on accessible elements.

Steps to Reproduce:
With UICollectionViewCell as WKWebview's superview: Activate VoiceOver, observe swipe navigation behavior.

With UIScrollview as WKWebview's superview: Activate VoiceOver, observe direct touch does not grant AccessibilityFocus.

Expected Results:
AccessibilityFocus should traverse all accessibility elements on screen. Direct touch on accessible element should grant accessibilityFocus.

Observed Results:
With UICollectionViewCell as WKWebview's superview: AccessibilityFocus does not exit WKWebview by swipe navigation.

With UIScrollview as WKWebview's superview: Direct touch on web view's accessible element does not grant focus.

Version:
iOS 9.3 / IPod Touch (5th Gen)
iOS 10.3 / Simulator

Notes:
Attempted workaround by inserting a UIView into the view hierarchy between the WKWebview and it's container view had no impact.

-
Product Version: 9, 10
Created: 2017-07-06T20:27:10.835980
Originated: 2017-06-23T00:00:00
Open Radar Link: http://www.openradar.me/32953541
